py/picca/pk1d/postproc_pk1d.py

```diff
@@ -749,22 +752,30 @@ def compute_average_pk_redshift(
                     standard_dev**2,
                     bounds=(0, np.inf),
                 )
+                coef_error, *_ = curve_fit(
+                    fitfunc_variance_pk1d,
+                    snr_bins,
+                    standard_dev_error**2,
+                    bounds=(0, np.inf),
+                )
 
                 # Model variance from fit function
                 data_snr[data_snr > MEANPK_FITRANGE_SNR[1]] = MEANPK_FITRANGE_SNR[1]
                 data_snr[data_snr < 1.01] = 1.01
                 variance_estimated = fitfunc_variance_pk1d(data_snr, *coef)
```

Comment:
is there any idea by now why a weight of `1/(snr-1)**2` is needed? and not of `1/snr**2 ~ ivar`

Reply:
Maybe ask @armengau on this point. This is a specificity of the SNR weighting for which only the SNR>1.01 are taken into account. I guess they tested with the snr - 1 method and gave closer P1D to the original measurement compare to just snr.

Reply:
yes, that's what I understood as well, just wondered if they found a theoretical reason why that should be the case in the meantime